src/evaluate.jl

```diff
@@ -54,7 +54,6 @@ computed values.
 function evaluate!(x::AbstractArray{Taylor1{T}}, δt::T,
```

Comment:
This case seems to already be covered by the following method?

Reply:
Good catch! Thanks. I'm removing this method.